wallet/wallet.c

```diff
@@ -1653,26 +1629,6 @@ wallet_payment_by_hash(const tal_t *ctx, struct wallet *wallet,
 	return payment;
 }
 
-struct secret *wallet_payment_get_secrets(const tal_t *ctx,
```

Comment:
Thanks for catching this, yes this function is no longer used as `path_secrets` is now part of the `struct payment`.